src/lib/Components/Bidding/Screens/BillingAddress.tsx

```diff
@@ -188,13 +208,25 @@ export class BillingAddress extends React.Component<BillingAddressProps, Billing
                 placeholder="Add state, province, or region"
                 onSubmitEditing={() => this.postalCode.root.focus()}
                 inputRef={el => (this.stateProvinceRegion = el)}
+                onFocus={() =>
+                  this.scrollView.scrollTo({
+                    x: 0,
+                    y: this.stateLayout.y - this.stateLayout.height - this.titleLayout.height,
+                  })
+                }
               />
 
               <StyledInput
                 {...this.defaultPropsForInput("postalCode")}
                 label="Postal code"
                 placeholder="Add your postal code"
                 onSubmitEditing={() => this.presentSelectCountry()}
+                onFocus={() =>
+                  this.scrollView.scrollTo({
```

Comment:
I wonder if we can be a little smarter about this, since we're repeating _nearly_ the same code each time. Something like a function named `yPosition(this.postalCodeLayout, this.titleLayout)` or similar? What do you think?